### PR TITLE
[NCL-8066] Add endpoints to retrieve DelAn Reports

### PR DESCRIPTION
**Pull request by janinko**
### Checklist:

* [ ] Have you added unit tests for your change?

### CODE REVIEW

File: rest-api/src/main/java/org/jboss/pnc/rest/api/endpoints/DeliverableAnalyzerReportEndpoint.java [hunk 1]
```diff
@@ -74,6 +78,57 @@ public interface DeliverableAnalyzerReportEndpoint {
...
+                            content = @Content(schema = @Schema(implementation = ErrorResponse.class))),
+                    @ApiResponse(
+                            responseCode = SERVER_ERROR_CODE,
+                            description = SERVER_ERROR_DESCRIPTION,
+                            content = @Content(schema = @Schema(implementation = ErrorResponse.class))) })
+    @GET
+    @TimedMetric
+    Page<DeliverableAnalyzerReport> getAll(@Valid @BeanParam PageParameters pageParams);
+
+    String GET_SPECIFIC_DESC = "Gets specific deliverable analyzer report.";
```
Comment (akridl) [inline rest-api/src/main/java/org/jboss/pnc/rest/api/endpoints/DeliverableAnalyzerReportEndpoint.java:108]: Move please to top of file, where are all other endpoints' descriptions

**Review by akridl — COMMENTED** [review]
Please, do finish also 3 already existing TODOs in DeliverableAnalyzerReportEndpointTest

File: rest-api/src/main/java/org/jboss/pnc/rest/api/endpoints/DeliverableAnalyzerReportEndpoint.java (hunk 1 above)
Comment (janinko) [reply rest-api/src/main/java/org/jboss/pnc/rest/api/endpoints/DeliverableAnalyzerReportEndpoint.java:108]: I'll do the exact oposite.

**Comment by akridl** [comment]
DeliverableAnalyzerReportEndpointTest, line 174, remove last remaining TODO please

File: rest-api/src/main/java/org/jboss/pnc/rest/api/endpoints/DeliverableAnalyzerReportEndpoint.java (hunk 1 above)
Comment (akridl) [reply rest-api/src/main/java/org/jboss/pnc/rest/api/endpoints/DeliverableAnalyzerReportEndpoint.java:100]: DeliverableAnalyzerReportEndpointTest, line 174, remove last remaining TODO please

**Review by akridl — APPROVED** [review]
<3

**Pull request merged**